x-pack/packages/kbn-cloud-security-posture/src/utils/utils.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { estypes } from '@elastic/elasticsearch';
+import { IndexDetails } from '@kbn/cloud-security-posture-common';
+
+export const getFindingsCountAggQuery = () => ({
+  count: { terms: { field: 'result.evaluation' } },
+});
+
+export const getFindingsCountAggQueryMisconfigurationPreview = () => ({
+  count: {
+    filters: {
+      other_bucket_key: 'other_messages',
```

Comment:
Looks like the `other_messages ` came from the docs. `unknown` would be a better name